### PR TITLE
BSM2 GUI Variable Name Update

### LINKED ISSUE #1153: BSM2 GUI Flowsheet
### Description

https://github.com/watertap-org/watertap/pull/1125 creates an initial GUI for the BSM2 flowsheet, but we eventually want to display the flowrate and compositions of all the streams in the "Outlet" tab, rather than just the two outlet streams exiting the flowsheet.

### Motivation

It will be useful for the project team to see the composition of each stream in the flowsheet rather than just the outlets.

### Possible Implementation

_No response_

### Additional Context

Priority

- [x] Dewatering liquid outlet
- [x] Translator block inlets and outlets
- [x] Anaerobic digestor gas outlet
- [x] Every other stream (with the possible exceptions of the storage tank and individual reactors in the ASP)
- [x] Increase resolution of flowsheet image

Adding a task here regarding parameter sweep results in the GUI
- [ ] add labels to distinguish between flow rate/concentration results across different streams of the flowsheet

Currently, one cannot tell which outputs are associated with specific streams in the flowsheet.

### PR DESCRIPTION
**Pull request by MarcusHolly**
## Fixes/Resolves:

Remaining checkbox in #1153 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by luohezhiming — APPROVED** [review]
LGTM

**Review by agarciadiego — APPROVED** [review]

**Review by adam-a-a — APPROVED** [review]
LGTM - @MichaelPesce @dangunter we realized that when looking at sweep results, if we weren't particular about naming the output, there could be confusion between multiple outputs with the same name. For example, a concentration at the outlet of one unit should be distinguished from the same component concentration at the outlet of a different unit in the flowsheet. 

TLDR: @MarcusHolly manually added category names in front of reported variable names to be able to distinguish between results. However, it might be better if this could be handled in the UI (i.e., for sweep results, display category and name for each entry).
